R/calculate_tdm.R

```diff
@@ -20,10 +20,12 @@
 #'   should be calculated.
 #'
 #' @return A tibble with the columns specified in the `additional_groups` input
-#'   as well as `tdm_technology_value`: the technology level transition disruption
-#'   metric, `tdm_sector_value`: the sector level transition disruption metric,
-#'   `tdm_portfolio_value`: the portfolio level transition disruption metric, as well
-#'   as `tdm_t0`, `tdm_delta_t1` and `tdm_delta_t2`: corresponding to the input
+#'   as well as `tdm_technology_value`: the technology level transition
+#'   disruption metric, `tdm_sector_value`: the sector level transition
+#'   disruption metric, `tdm_portfolio_value`: the portfolio level transition
+#'   disruption metric, `tdm_metric`: a flag that indicates what each `*_value`
+#'   is being calculated against (e.g. "portfolio" or "scenario"),  as well as
```

Comment:
I know the wording here is a bit awkward, if you can think of a clearer way to explain the purpose of that column, that would be cool.

Reply:
should it say

> what each `tdm_*_value` is being calculated against

?

Reply:
👍